constants/additionalChainRegistry/chainid-2107.js

```diff
@@ -0,0 +1,27 @@
+export const data = {
+    "name": "International Bitcoin Virtual Machine Testnet",
+    "chain": "IBVM Testnet",
+    "icon": "ibvmtest",
+    "rpc": [
+      "https://rpc-testnet.ibvm.io/",
+      "wss://rpc-testnet.ibvm.io/ws/"
+    ],
+    "faucets": ["https://faucet.ibvm.io"],
+    "nativeCurrency": {
+      "name": "IBVM Bitcoin",
+      "symbol": "BTC",
+      "decimals": 18
+    },
+    "infoURL": "https://ibvm.io/",
+    "shortName": "IBVMT",
```

Comment:
add the t for testnet
Once mainnet launch you can use IBVM as the shortname (on the mainnet)